formspree/templates/static_pages/index.html

```diff
@@ -88,7 +88,7 @@ <h4>Who are you people?</h4>
             <div class="col-1-2">
                 <div class="card">
                     <h4>What about privacy?</h4>
-                    <p>We only store contents of the last 100 form submissions and delete the rest. Emails are sent using SendGrid's API, so on that end their privacy policies apply.</p>
+                    <p>Unless you've opted in, we only store contents of the last {{ config.MONTHLY_SUBMISSIONS_LIMIT }} form submissions and delete the rest. Emails are sent using SendGrid's API, so on that end their privacy policies apply.</p>
```

Comment:
`We only store up to {{ config.ARCHIVE_SUGMISSIONS_LIMIT }} form submissions per month. For details on our privacy practices, please see our <a href="????">privacy policy</a>.`